int32 length
char[] pcm_buf
